geotrek/api/tests/test_v2.py

```diff
@@ -428,6 +436,18 @@ def setUpTestData(cls):
         cls.site2.themes.add(cls.theme3)
         cls.label_3 = common_factory.LabelFactory()
         cls.site2.labels.add(cls.label_3)
+        cls.hdviewpoint_trek = common_factory.HDViewPointFactory(
+            content_type=ContentType.objects.get_for_model(trek_models.Trek),
+            object_id=cls.treks[0].pk
```

Comment:
as it's a generic foreign, you can just set:
```suggestion
            content_object=cls.treks[0]
```
instead of set contnt_type and object_id